themes/digital.gov/layouts/communities/single.html

```diff
@@ -1,154 +1,129 @@
-                        <p>{{- $e.terms | markdownify -}}</p>
-                        {{- else -}}
-                        <p>Open to the public.</p>
-                        {{- end -}}
-                      {{- end -}}
-
-                      {{/* GOVT */}}
-                      {{- if eq $e.type "government" -}}
-                        {{- if $e.terms -}}
-                        <p>{{- $e.terms | markdownify -}}</p>
-                        {{- else -}}
-                        <p>Open to anyone with a .gov or .mil email address.</p>
-                        {{- end -}}
-                      {{- end -}}
-
-                      {{/* GOVT_ONLY */}}
-                      {{- if eq $e.type "government_only" -}}
-                        {{- if $e.terms -}}
-                        <p>{{- $e.terms | markdownify -}}</p>
-                        {{- else -}}
-                        <p>Open to any government employee.</p>
-                        {{- end -}}
-                      {{- end -}}
-                    </div>
-
-                    <div class="join-community-form__body-email">
-                      {{/*
-                        Community JOIN types ---
-                        - Listerv via an email (most common)
-                        - Listerv via a form
-                        - Google Group
-                        - Slack
-                      */}}
-                      {{/* Listserv Platforms without form sign-up option */}}
-                      {{- if and (eq $e.platform "listserv") (not $e.subscribe_form ) -}}
-                        {{- partial "core/community-join-without-form.html" (dict "context" . "e" $e) -}}
-                      {{- end -}}
-
-                      {{/* Google Group platforms */}}
-                      {{- if and (eq $e.platform "google-group") ( $e.subscribe_form ) -}}
-                        {{- partial "core/community-join-google-group.html" (dict "context" . "e" $e) -}}
-                      {{- end -}}
-
-                      {{/* Slack platforms */}}
-                      {{- if and (eq $e.platform "slack") ( $e.subscribe_form ) -}}
-                        {{- partial "core/community-join-slack.html" (dict "context" . "e" $e) -}}
-                      {{- end -}}
-
-                    </div>
-                  </div>
-                {{- end -}}
-
-              </section>
-              {{- end -}}
-              <div>
-                {{- .Content -}}
-              </div>
-
-              <div class="community_events">
-                {{- partial "core/get-upcomingevents.html" . -}}
-              </div>
-
-              <div class="Community_conduct">
-                <h2 id="community-conduct">Community Conduct</h2>
-                {{- partial "core/community_pagefooter.html" . -}}
-                <br>
+    <div class="usa-in-page-nav-container">
+      <aside class="usa-in-page-nav" data-title-heading-level="h3" data-root-margin="-350px 0px -350px 0px"></aside>
```

Comment:
Can we add a comment for what these `data-root-margin` values are doing?

Reply:
Good idea, I've added comment explaining why these values are set.